test/e2e/windows/host_process.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package windows
+
+import (
+	"context"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] HostProcess containers", func() {
+	ginkgo.BeforeEach(func() {
+		e2eskipper.SkipUnlessNodeOSDistroIs("windows")
+	})
+
+	f := framework.NewDefaultFramework("host-process-test-windows")
+
+	ginkgo.It("should run as a process on the host/node", func() {
+
+		ginkgo.By("selecting a Windows node")
```

Comment:
Add 
```
defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.WindowsHostProcessContainers, true)()
```

Reply:
or some other way to validate that the feature gate was actually set for the test

Reply:
For the Windows E2E tests we manage the test configs at https://github.com/kubernetes-sigs/windows-testing/tree/master/job-templates and the test configs specify what feature gates are used.

We've got a separate issue for the Windows E2E tests where we don't have a good way to restrict test cases to only clusters with features enabled but we're working to solve that too.

Reply:
you can try something similar to this: https://github.com/kubernetes/kubernetes/pull/99612/files#diff-471e8786474b22c41fa2d0ca2a9c3950a7da2e0ac7ab6de638c505204b39bf8dR142